ginga/util/io_fits.py

```diff
@@ -491,7 +543,7 @@ def __init__(self, logger):
         self.hdu_info = []
         self.hdu_db = {}
 
-    def fromHDU(self, hdu, ahdr):
```

Comment:
Might want to deprecate this and have it return `copy_header(...)` instead of outright removal.